src/samsung.ts

```diff
@@ -522,9 +522,8 @@ class Samsung {
 
   private _saveTokenToFile(token: string) {
     try {
-      fs.accessSync(this.TOKEN_FILE, fs.constants.F_OK)
```

Comment:
When the token.txt is not present this call causes the exception which results in the token not getting saved.